spec/routines/search_users_spec.rb

```diff
@@ -124,4 +124,4 @@
 
   end
 
-end
\ No newline at end of file
+end
```

Comment:
I think your solution is better.  I've changed it to `email_addresses.order(:value).first` :)